### PR TITLE
test(index): add test showing error

### PR DESCRIPTION
**Pull request by bassettsj**
Found a bug in the last release, it looks like there is a broken path. I wanted to add unit testing too to hopefully catch other issues later. :-)

```
Cannot find module './src/api/test_profile' from 'index.js'

      14 | const Snapshot = require('./src/api/snapshot')
      15 | const Test = require('./src/api/test')
    > 16 | const TestProfile = require('./src/api/test_profile')
      17 | const Agent = require('./src/api/location')
      18 |
      19 | module.exports = {
```

### CODE REVIEW

File: __tests__/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Index should be able to resolve the same dependencies 1`] = `
```
Comment (bassettsj) [inline __tests__/__snapshots__/index.test.js.snap]: Just to capture what the API should be like will be helpful for catching the public API changes.

File: index.js [hunk 2]
```diff
@@ -1,19 +1,13 @@
...
 
 if (!process.env.CALIBRE_API_TOKEN) {
-  console.log(
-    chalk.grey(
-      'Please set CALIBRE_API_TOKEN as an environment variable. Create your token via the team API tokens page, or see calibreapp.com/docs/api/tokens for help.'
-    ),
-    chalk.blue(`${process.env.CALIBRE_HOST}/home`)
-  )
-  process.exit()
+  throw new Error('Please set CALIBRE_API_TOKEN as an environment variable. Create your token via the team API tokens page, or see calibreapp.com/docs/api/tokens for help.');
```
Comment (bassettsj) [inline index.js]: The `process.exit` will cause failures to not be reported in jest, this also seems more like an error to me honestly and should fail hard.

Comment (benschwarz) [reply index.js]: Will it cleanly exit?

Comment (bassettsj) [reply index.js]: It will exit with an error code, yes.

File: index.js [hunk 3]
```diff
@@ -1,19 +1,13 @@
...
 }
 
 const Site = require('./src/api/site')
 const Snapshot = require('./src/api/snapshot')
 const Test = require('./src/api/test')
-const TestProfile = require('./src/api/test_profile')
+const TestProfile = require('./src/api/test-profile')
```
Comment (benschwarz) [inline index.js]: 🤦‍♂️

**Comment by bassettsj** [comment]
@benschwarz The only other thing I could think would be maybe selecting a code style and I could make sure my changes work with eslint or something?

**Comment by bassettsj** [comment]
@benschwarz  Maybe it is good enough for now. 😄 We can add that later.

**Comment by benschwarz** [comment]
Hey @bassettsj, I've enabled codeclimate for the repo, and will introduce #15 soon too.

**Comment by benschwarz** [comment]
Thanks so much for working through those last issues @bassettsj, I didn't have time to finish it off yesterday. Merging. Thank you for all your contributions here!

**Pull request merged**